docs-starlight/public/robots.txt

```diff
@@ -0,0 +1 @@
+User-agent: * Disallow: /
```

Comment:
Is this only for test deployment?

Reply:
Ya. I left a todo to remove it after we replace terragrunt.gruntwork.io with terragrunt-v1.grubtwork.io.